src/Microsoft.Security.Utilities.Core/LowConfidenceClassifiers/SEC101_060_LooseSasSecret.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System;
+using System.Collections.Generic;
+using System.Web;
+
+namespace Microsoft.Security.Utilities
+{
+    public class LooseSasSecret : RegexPattern
+    {
+        public LooseSasSecret()
+        {
+            Id = "SEC101/060";
+            Name = nameof(LooseSasSecret);
+            DetectionMetadata = DetectionMetadata.HighEntropy;
+            Pattern = @$"(?i)(?:^|[?;&])(?:dsas_secret|sig)=(?<refine>[0-9a-z\/+%]{{43,129}}(?:=|%3d))";
+            Signatures = new HashSet<string>(new[] { "sig=", "ret=" });
+        }
+
+
+        // HttpUtility.UrlEncode(new string('/', 43));
+        private const string FortyThreeBase64EncodedSlashes = "%2f%2f%2f%2f%2f%2f%2f%2f%2f%2f%2f%2f%2f%2f%2f%2f%2f%2f%2f%2f%2f%2f%2f%2f%2f%2f%2f%2f%2f%2f%2f%2f%2f%2f%2f%2f%2f%2f%2f%2f%2f%2f%2f";
```

Comment:
This is hard to maintain---what if someone mistakenly nukes a %2f? This only seems to be used in generating examples. Can't we add a helper method that creates 43 %2f on the fly and returns it? Is this string on a perf sensitive path?

Reply:
I wouldn't provide a helper because this is a const value for a contrived case. There is a code comment that describes how it was produced. 

If someone nukes a %2f that still matches the pattern. Why would a use go tinker with this exactly? I assume they will put that change through code review? If there was a helper that, for example, constructed this string, what if someone mistakenly changed that code? Just trying to understand what you're defending against. :)